doc/cug.tex

```diff
@@ -1400,12 +1400,62 @@ \subsection{Remote Tasks}
 ' "/opt/cylc/bin/cylc" '"'"'job-submit'"'"\
 ' --remote-mode "/home/oliverh/cylc-run/tut.oneoff.remote/log/job/1/hello/01/job"'
 \end{lstlisting}
-(Don't be intimated by this - it really quite straightforward and would appear
-much simpler if the job log path was shorter!). Remote task job logs are
+(Don't be intimated by this - it is really quite straightforward and would
+appear much simpler if the job log path was shorter!). Remote task job logs are
 saved to the suite run directory on the task host, not on the suite host,
-although they can be retrieved by right-clicking on the task in the GUI. Rose
-(section~\ref{Rose}) provides a task event handler to pull logs back to
-the suite host.
+although they can be retrieved by right-clicking on the task in the GUI. If you
+want the job logs pulled back to the suite host automatically, you can set
+\lstinline@[[[remote]]]retrieve job log=True@:
+
+\lstset{language=suiterc}
+\begin{lstlisting}
+[runtime]
+    [[hello]]
+        script = "sleep 10; echo Hello World!"
+        [[[remote]]]
+            host = server1.niwa.co.nz
+            retrieve job log = True
+\end{lstlisting}
+
+The suite will attempt to \lstinline=rsync= the job logs once from the remote
+host each time a task job completes. E.g. if the job file is
+\lstinline=~/cylc-run/tut.oneoff.remote/log/job/1/hello/01/job=, anything under
+\lstinline=~/cylc-run/tut.oneoff.remote/log/job/1/hello/01/= will be retrieved.
+
+Some batch systems have considerable delays between the time when the job
+completes and when it writes the job logs in its normal location. If this is
+the case, you can configure an initial delay and retry delays for job log
+retrieval by setting some \lstinline=[[[remote]]]retrieve job log delays=. E.g.:
+
+\lstset{language=suiterc}
+\begin{lstlisting}
+[runtime]
+    [[hello]]
+        script = "sleep 10; echo Hello World!"
+        [[[remote]]]
+            host = server1.niwa.co.nz
+            retrieve job log = True
+            # Retry after 10 seconds, 1 minute and 3 minutes
+            retrieve job log delays = PT10S, PT1M, PT3M
+\end{lstlisting}
+
+Finally, if the disk space of the suite host is limited, you may want to
+restrict the maximum sizes of the job log files to retrieve by setting
+\lstinline@[[[remote]]]retrieve job log max size=SIZE@. The value of SIZE can
+be anything that is acceptable by the \lstinline@--max-size=SIZE@ option of the
+\lstinline=rsync= command. E.g.:
+
+\lstset{language=suiterc}
+\begin{lstlisting}
+[runtime]
+    [[hello]]
+        script = "sleep 10; echo Hello World!"
+        [[[remote]]]
+            host = server1.niwa.co.nz
+            retrieve job log = True
+            # Don't get anything bigger than 10MB
+            retrieve job log max size = 10M
```

Comment:
(does rsync report anything if this happens, so we can log the reason for a missing file?)